lib/y2lan_utils.pm

```diff
@@ -0,0 +1,74 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Useful subroutines for yast2_lan related test modules
+# Maintainer: Veronika Svecova <vsvecova@suse.com>
+
+package y2lan_utils;
```

Comment:
We have some part already in `lib\y2lan_restart_common`. I like the name you've introduced more, as there are not so many imports, maybe we should combine those.

Reply:
Ok, that makes sense. After consulting it with @jknphy (as the maintainer of `y2lan_restart_common`), I'll go ahead and combine them.